docs/writing-stories/args.md

```diff
@@ -76,6 +76,20 @@ You can also define args at the component level; they will apply to all the comp
 
 <!-- prettier-ignore-end -->
 
+## Project args
```

Comment:
I personally like the clarity of "project" rather than "global" here, but we currently use the term "global" for [parameters](https://storybook.js.org/docs/react/writing-stories/parameters#global-parameters) and [decorators](https://storybook.js.org/docs/react/writing-stories/decorators#global-decorators).

Reply:
Yes! We changed this internally in 6.4, as it was super confusing to have "global args" and "globals" (which once upon a time were actually called `globalArgs` haha).

We should transition to using project everywhere I think. What's your recommendation in the meantime?

Reply:
I think this PR should use "global". Then we can make another PR to update it to "project" everywhere, possibly only including that change in `6.5`+.